README.md

```diff
@@ -33,26 +33,56 @@ For [conda](https://docs.conda.io/) and [spack](https://spack.io/) installation
 
 ## Documentation
 
-You can find the documentation for torchgeo on [ReadTheDocs](https://torchgeo.readthedocs.io).
+You can find the documentation for TorchGeo on [ReadTheDocs](https://torchgeo.readthedocs.io).
 
-## Example usage
+## Example Usage
 
-The following sections give basic examples of what you can do with torchgeo. For more examples, check out our [tutorials](https://torchgeo.readthedocs.io/en/latest/tutorials/getting_started.html).
+The following sections give basic examples of what you can do with TorchGeo. For more examples, check out our [tutorials](https://torchgeo.readthedocs.io/en/latest/tutorials/getting_started.html).
 
-### Train and test models using our PyTorch Lightning based training script
+First we'll import various classes and functions used in the following sections:
 
-We provide a script, `train.py` for training models using a subset of the datasets. We do this with the PyTorch Lightning `LightningModule`s and `LightningDataModule`s implemented under the `torchgeo.trainers` namespace.
-The `train.py` script is configurable via the command line and/or via YAML configuration files. See the [conf/](conf/) directory for example configuration files that can be customized for different training runs.
+```python
+from torch.utils.data import DataLoader
+from torchgeo.datasets import CDL, Landsat7, Landsat8, TropicalCycloneWindEstimation, stack_samples
+from torchgeo.samplers import RandomGeoSampler
+```
 
-```console
-$ python train.py config_file=conf/landcoverai.yaml
+### Composing geospatial datasets
+
+Many use cases involve working with geospatial data and combining it in intelligent ways. In this example, we assume that the user has Landsat 7 and 8 imagery downloaded. We first create a single dataset containing all Landsat imagery. We only use the Landsat 7 bands for Landsat 8 and take the union between both datasets.
```

Comment:
Can you expand on the intro to this section a little bit? My thought reading this is, "Many use cases of what?"